docs/development/forked-pr.md

```diff
@@ -23,26 +23,22 @@ Look for the green check mark in the PR.
 
 ## Reviewing PRs from Forked Repos
 
-Upon submission of a pull request, the Cumulus development team will review the code
+Upon submission of a pull request, the Cumulus development team will review the code.
 
-The request will then either be merged, declined, or an adjustment to the code will be requested
+The request will then either be merged, declined, or an adjustment to the code will be requested.
 
 PRs from forked repos cannot directly merged to master. Cumulus reviews must follow the following steps before completing the review process:
 
 1. Create a new branch:
 
-     $ git checkout -b from-from-<name-of-the-branch> master
+     $ git checkout -b from-<name-of-the-branch> master
 
-2. Pull in the forked branch
+1. Push the new branch to github
```

Comment:
Shouldn't this be step `2`? All the steps are numbered `1`.

Reply:
This is a markdown trick (I can't find a reference) that allows you to number all of your items as 1 but it will output the correct number for each item. But it's probably clearer to just number things correctly in the source code. I'll fix.

Reply:
Not sure I like that trick.. I thought the whole point of Markdown is being readable raw. I know it's a trivial example but relying on markdown parsers to make things intelligible doesn't seem like a great idea.